### DISCUSSION TITLE
Add GitHub discussions

### DISCUSSION THREAD

**Discussion by henryiii** [pypa/cibuildwheel#480] [Ideas]
I think enabling discussions (added for everyone today) could be really useful. Especially the show and tell category would be interesting. You can move issues into discussions.

https://github.blog/2020-12-08-new-from-universe-2020-dark-mode-github-sponsors-for-companies-and-more/

**Comment by joerick** [comment]
Sounds good! Yes that could be useful for the more support-oriented issues we get, and the open-ended long-running discussions. I'll enable it and we can see what we think.

**Comment by joerick** [comment]
And here we are. I suppose there's a job to migrate relevant issues over and decide on categories, I'm thinking maybe just `General` and `Q&A`.

**Reply by henryiii** [reply]
Can you disable or add categories? I think they are customized - that is, Q&A has the idea of an answer, etc., and I didn't think you could turn any off quite yet. The starting set seems to be pretty useful, though - General for announcements, Ideas for new feature requests, Q&A for help, and Show & Tell for links to great community examples.

**Reply by YannickJadoul** [reply]
Let's see what this does, already? Guess it'll take everyone some time to adapt? :-)

**Reply by henryiii** [reply]
Nevermind, you can edit the categories, and answers enabled is a setting!

**Comment by henryiii** [comment]
Adding a message that I'll mark as outdated.

**Comment by henryiii** [comment]
Adding a message that I'll mark as resolved.

**Comment by henryiii** [comment]
I don't know why you can hide with "resolved" or "outdated" but it seems to make no difference. I'd expect it to say something different for each reason, not just "This comment has been hidden".

**Reply by YannickJadoul** [reply]
Not sure I'm a huge fan of that; seeing that makes me think it was spam or so :-/

**Reply by henryiii** [reply]
If the message was better it would be fine. But not in its current form.

**Reply by henryiii** [reply]
Or if there was any indication of the reason at all......

**Reply by YannickJadoul** [reply]
Agreed :-)

**Comment by henryiii** [comment]
There seems to be a voting system for discussions now.

**Comment by henryiii** [comment]
Not very related to anything, and there's no "chat" like gitter, so I'll put this here. My two posts over cibuildwheel (the [new features post](https://iscinumpy.gitlab.io/post/cibuildwheel-1-9-0/), and [an overview](https://iscinumpy.gitlab.io/post/overview-of-cibuildwheel/) since I haven't done one before) are up. I'll be linking to those in our SciPy proposal.

**Reply by YannickJadoul** [reply]
Nice read! I was thinking "should we link to this?", but then again, anything that's in here should probably also be in our docs, if it's not yet the case? It will hopefully be great for library authors to discover, though! :-)

Quick thing: these two links are broken: `[Tox][] or [Nox][]`

**Reply by henryiii** [reply]
Thanks, by the way, I've fixed this. :)

**Reply by joerick** [reply]
These are really nice posts, @henryiii !
